### PR TITLE
Added ESlint and Travis config

### PR DESCRIPTION
**Pull request by eXistenZNL**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+    - "10"
```
Comment (RickvdStaaij) [inline .travis.yml]: Must it really be a string? :innocent:

**Comment by RickvdStaaij** [comment]
Also to fix the cypress errors:

```
{
    "env": {
        "cypress/globals": true
    },
    "plugins": [
        "cypress"
    ],
```

File: .travis.yml (hunk 1 above)
Comment (eXistenZNL) [reply .travis.yml]: I believe so: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/

**Comment by RickvdStaaij** [comment]
Y U NO FORK? I was already questioning the 2 builds :innocent:

**Comment by RickvdStaaij** [comment]
Only a couple of errors left, which sound fair to me!

File: .travis.yml (hunk 1 above)
Comment (eXistenZNL) [reply .travis.yml]: 1d374b14141689131240501455_700wa_0

**Comment by eXistenZNL** [comment]
I totally pushed to upstream instead of origin, my bad! Will drink some more coffee before devving next time :+1:

**Pull request merged**